File: C++/1416.cpp
#include <iostream>

using namespace std;

int main(){
	int n, arr[100], len = 0;
	cin >> n;
	
	if(n==0)
		cout << 0;
	else{
		for(int i = 0;;i++){
			if(n==0) break;
			arr[i] = n%2;
			len++;
			n /= 2;
		}
		for(int j = len - 1; j >= 0; j--)
			cout << arr[j];
	}
}

File: Java/1416.java
import java.util.Scanner;
import java.util.StringTokenizer;

public class Main{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		int arr[] = new int[100];
		int len = 0;
		
		if(n==0)
			System.out.println(0);
		else {
			for(int i = 0;;i++){
				if(n==0) break;
				arr[i] = n%2;
				len++;
				n /= 2;
			}
			for(int j = len - 1; j >= 0; j--)
				System.out.printf("%d", arr[j]);
		}
	}
}